client/blocks/eligibility-warnings/utils.ts

```diff
@@ -0,0 +1,3 @@
+export function isAtomicSiteWithoutBusinessPlan( holds: string[] ): boolean {
+	return holds.includes( 'TRANSFER_ALREADY_EXISTS' ) && holds.includes( 'NO_BUSINESS_PLAN' );
```

Comment:
Shouldn't we also check for `NO_ECOM_PLAN`?

Reply:
There is no `NO_ECOM_PLAN`. If a site is on Atomic but is on a plan below Business it returns `NO_BUSINESS_PLAN` which means it needs to be on _at least_ a business plan.